apps/storefront/src/shared/service/request/base.ts

```diff
@@ -1,6 +1,18 @@
-const { VITE_B2B_URL } = import.meta.env;
+import { Environment, EnvSpecificConfig } from '@/types';
 
-const B2B_BASIC_URL = VITE_B2B_URL;
+const ENVIRONMENT_B2B_API_URL: EnvSpecificConfig<string> = {
+  local: import.meta.env.VITE_B2B_URL ?? 'http://localhost:9000',
+  integration: 'https://api-b2b.integration.zone',
+  staging: 'https://api-b2b.staging.zone',
+  production: 'https://api-b2b.bigcommerce.com',
+};
+
+function getAPIBaseURL() {
+  const environment: Environment = window.B3.setting.environment ?? Environment.Production;
+  return ENVIRONMENT_B2B_API_URL[environment];
+}
```

Comment:
This function it's creating problems to run the project locally

Where should I configure this value?
Run it with this branch
<img width="790" alt="image" src="https://github.com/user-attachments/assets/27db8a52-e1ed-4f6f-bbdb-25e4abb51df8">

Modify the branch
![image](https://github.com/user-attachments/assets/d6d571ca-e8c3-416b-93ee-66eafee9acdf)
<img width="1482" alt="image" src="https://github.com/user-attachments/assets/7f481f69-6f3b-40f1-a0b8-a34adc45476d">

Reply:
Ah good point - local needs to have this set to Local.
I'll add a `VITE_ENVIRONMENT` for local

Reply:
I made it default to `Environment.Local` so that should help.